samples/model/README.md

```diff
@@ -0,0 +1,48 @@
+# Bring Your Own Container Sample
+
+This sample demonstrates how to start model jobs using your own model script, packaged in a SageMaker-compatible container, using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.                     
+
+## Prerequisites
+
+This sample assumes that you have already configured an EKS cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](TODO).
+
+In order to follow this script, you must first create a model script packaged in a Dockerfile that is [compatible with Amazon SageMaker](https://docs.aws.amazon.com/sagemaker/latest/dg/amazon-sagemaker-containers.html). 
+
+## Preparing the modelTuning Script
+
+### Uploading your Script
+
+All SageMaker model jobs are run from within a container with all necessary dependencies and modules pre-installed and with the model scripts referencing the acceptable input and output directories. This container should be uploaded to an [ECR repository](https://aws.amazon.com/ecr/) accessible from within your AWS account. When uploaded correctly, you should have a repository URL and tag associated with the container image - this will be needed for the next step. Sample container images are [available](https://docs.aws.amazon.com/sagemaker/latest/dg/ecr-us-west-2.html).
```

Comment:
I think this should be rewritten for model. Models are not one-time jobs running "within a container", but are instead a specification for how an endpoint would mount a container and where it passes inputs and outputs (and in which format).